### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by xSimpLUL** [danthe1st/Mee6-bypasser#29] [Support] [answered]
Users aren't getting their roles when typing !rank. Bot has all perms and is above all other roles, and the role is configured properly with mee6 bypasser and leaderboards are public.

**Comment by danthe1st** [comment, marked as answer]
Mee6-bypasser has a rate limit that allows only one user per server to use it per minute.

This is in place so that Mee6-bypasser is not getting blocked from Mee6 because of sending too many requests.

If you want to remove this rate limit, clone the bot, remove it from the source code, build it and host Mee6-bypasser by yourself.

## [Edit]

This ratelimit has been subsitentially lowered, Me(e)-Bypasser does not send requests to Mee6 anymore and should not be a problem any more.
